network/tailscale/vars.yaml

```diff
@@ -0,0 +1 @@
+VERSION: "{{ .TAILSCALE_VERSION }}-{{ .BUILD_ARG_TAG }}"
```

Comment:
i think just the tailscale version is fine. The `BUILD_ARG_TAG` is only used when the talos version has to match the extension version (eg: in case of kernel modules)

Reply:
Fixed